lib/subordinateio.c

```diff
@@ -685,6 +686,41 @@ uid_t sub_uid_find_free_range(uid_t min, uid_t max, unsigned long count)
 	return start == ULONG_MAX ? (uid_t) -1 : start;
 }
 
+
+/*
+ * bool want_subuid_file(void)
+ *
+ * Checks if /etc/subuid is in use.
+  *
+ * Returns true if /etc/subuid should be opened/created, if
+ * false is returned, /etc/subuid should not be accessed.
```

Comment:
There's an alignment issue in line 694.

BTW, what do you mean by "Checks if /etc/subuid is in use"?  In use by whom?